src/requisition/requisition.js

```diff
@@ -395,6 +395,14 @@
             return deferred.promise;
         }
 
+        function updateRequisition(requisition, updated) {
+            var availableOffline = requisition.$availableOffline;
+
+            requisition.status = updated.status;
+            requisition.statusChanges = updated.statusChanges;
```

Comment:
I'm a little concerned by the manual modification of the requisition object stored in the local storage. I remember us wanting the server response to be saved to make sure we've correct requisition stored in the local storage.